File: src/Easy/44-Fizzbuzz/Nim/main.nim
const limit = 100

for i in 1 .. limit:
  echo if i mod 3 == 0 and i mod 5 == 0:
    "FizzBuzz"
  elif i mod 3 == 0:
    "Fizz"
  elif i mod 5 == 0:
    "Buzz"
  else:
    $i

File: src/Easy/44-Fizzbuzz/Haskell/Main.hs
import Data.Foldable

fizzBuzz :: Int -> String
fizzBuzz x
 | x `mod` 15 == 0 = "FizzBuzz"
 | x `mod` 5 == 0 = "Buzz"
 | x `mod` 3 == 0 = "Fizz"
 | otherwise = show x

main :: IO ()
main = for_ [1..100] (putStrLn . fizzBuzz)
-- can also be main = traverse_ (putStrLn . fizzBuzz) [1..100]
